CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
   and change the type of the reported exception to `\Sentry\Exception\FatalErrorException` (#788)
 - Add a static factory method to create a breadcrumb from an array of data (#798)
 - Add support for `SENTRY_ENVRIONMENT` and `SENTRY_RELEASE` environment variables (#810)
+- Implement class serializers and interface to customize how objects are serialized (#809)
```

Comment:
I think a better description would be `Add the `class_serializers` option to make it possible to customize how objects are serialized in the event payload`